src/main/java/rs/hybridit/dto/BookDto.java

```diff
@@ -0,0 +1,106 @@
+package rs.hybridit.dto;
+
+import java.util.HashSet;
+import java.util.Set;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import org.hibernate.validator.constraints.ISBN;
+import rs.hybridit.model.BookCopy;
+
+public class BookDto {
+
+	private Long id;
+
+	@ISBN
+	private String isbn;
+
+	@Size(min = 3, max = 15, message="Length of name must be between 3 and 15.")
+	@NotBlank(message = "Name is not allowed to be empty.")
```

Comment:
not blank already verifies object presence

Reply:
Basically, not black does a null check and size check, so in this case only @NotBlank is needed (without @NotNull)